### PR TITLE
[DM-3594] OPC-UA reverse scanner NPE fix - changelog

### PR DESCRIPTION
**Pull request by jakubsobolewskisag**
OPC-UA changelog

### CODE REVIEW

File: content/change-logs/device-management/opcua-10-20-82-0-fix-reverse-scan-edge-case.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-Tf05_KQ-B
+    label: OPC UA
+build_artifact:
+  - value: tc-MLn0oFRX-
+    label: opcua
+ticket: DM-3594
+version: 10.20.82.0
+---
+In previous OPC-UA Gateway versions it was possible to get a NullPointerException during reverse address space scan in rare cases when the last processed node was not a root node.
```
Comment (Oezge-Akin) [inline content/change-logs/device-management/opcua-10-20-82-0-fix-reverse-scan-edge-case.md:17]: This desc. with NullPointerException and etc. is very technical. It should be from user perspective. What was not working? Also with this tasks 2 issues are resolved one was relevant to full scan.

Comment (jakubsobolewskisag) [reply content/change-logs/device-management/opcua-10-20-82-0-fix-reverse-scan-edge-case.md]: done

File: content/change-logs/device-management/opcua-10-20-82-0-fix-reverse-scan-edge-case.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+component:
+  - value: component-Tf05_KQ-B
+    label: OPC UA
+build_artifact:
+  - value: tc-MLn0oFRX-
+    label: opcua
+ticket: DM-3594
+version: 10.20.82.0
+---
+In earlier versions of the OPC-UA Gateway, there was a rare issue where a reverse address space scan could unexpectedly
```
Comment (BeateRixen) [inline content/change-logs/device-management/opcua-10-20-82-0-fix-reverse-scan-edge-case.md:17]:
```suggestion
In earlier versions of the OPC UA gateway, it rarely happened that a reverse address space scan unexpectedly 
```

File: content/change-logs/device-management/opcua-10-20-82-0-fix-reverse-scan-edge-case.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  - value: component-Tf05_KQ-B
+    label: OPC UA
+build_artifact:
+  - value: tc-MLn0oFRX-
+    label: opcua
+ticket: DM-3594
+version: 10.20.82.0
+---
+In earlier versions of the OPC-UA Gateway, there was a rare issue where a reverse address space scan could unexpectedly 
+fail if the last node processed wasn't a root node. We've now fixed this problem. Also, if a scan encounters an issue
```
Comment (BeateRixen) [inline content/change-logs/device-management/opcua-10-20-82-0-fix-reverse-scan-edge-case.md:18]:
```suggestion
failed if the last node processed wasn't a root node. This issue has been fixed. Also, if a scan encounters an issue 
```

File: content/change-logs/device-management/opcua-10-20-82-0-fix-reverse-scan-edge-case.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
+---
+date:
+title: Fixed OPC-UA address space scanning failures in known edge cases
```
Comment (BeateRixen) [inline content/change-logs/device-management/opcua-10-20-82-0-fix-reverse-scan-edge-case.md:3]:
```suggestion
title: Fixed issue with OPC UA address space scanning
```

**Review by BeateRixen — APPROVED** [review]

**Review by Oezge-Akin — APPROVED** [review]

**Pull request merged**